File: login/src/main/java/com/klcw/app/login/bean/CommonList.java
package com.klcw.app.login.bean;

import java.util.List;

/**
 * @Auther: yd
 * @datetime: 2018/11/5
 * @desc:
 */
public class CommonList {

    public List<CommonBean> data;
    public int errorCode;
    public String errorMsg;

    @Override
    public String toString() {
        return "CommonList{" +
                "data=" + data +
                ", errorCode=" + errorCode +
                ", errorMsg='" + errorMsg + '\'' +
                '}';
    }
}

File: login/src/main/java/com/kk/app/login/bean/CommonList.kt
package com.kk.app.login.bean;

import java.util.List;

/**
 * @author kk
 * @datetime: 2018/10/24
 * @desc:
 */
public class CommonList {

    public List<CommonBean> data;
    public int errorCode;
    public String errorMsg;

    @Override
    public String toString() {
        return "CommonList{" +
                "data=" + data +
                ", errorCode=" + errorCode +
                ", errorMsg='" + errorMsg + '\'' +
                '}';
    }
}
